### DISCUSSION TITLE
Hololens 1 - Technology Stack

### DISCUSSION THREAD

**Discussion by loibnerg** [MixedRealityToolkit/MixedRealityToolkit-Unity#547] [Q&A]
As I am still using the Hololens 1 in an educational environment to show the capabilities of Mixed Reality Hardware, what would be the recommended TechStack for it? Currently I am still sticking to Unity 2019 with MRTK 2.8. Should I try to upgrade to Unity 2021 and MRTK 3 or is it not worth a try? Thanks in advance.

**Comment by AMollis** [comment]
MRTK3 unfortunately does not support HoloLens 1.   MRTK3 requires OpenXR, which HL1 does not support.  You'll need to stick with a Unity version that supports the older Windows Mixed Reality APIs and MRTK2.
